examples/flax/language-modeling/README.md

```diff
@@ -33,11 +33,37 @@ in Norwegian on a single TPUv3-8 pod.
 
 The example script uses the 🤗 Datasets library. You can easily customize them to your needs if you need extra processing on your datasets.
 
-Let's start by creating a folder to save the trained model and a symbolic link to the `run_mlm_flax.py` script.
+Let's start by creating a model repository to save the trained model and logs.
+Here we call the model `"norwegian-roberta-base"`, but you can change the model name as you like.
+
+You can do this either directly on [huggingface.co](https://huggingface.co/new) (assuming that
+you are logged in) or via the command line:
+
+``
+huggingface-cli repo create norwegian-roberta-base
+```
+
+Next we clone the model repository to add the tokenizer and model files.
+
+```
+git clone https://huggingface.co/<your-username>/norwegian-roberta-base
+```
+
+To ensure that all tensorboard traces will be uploaded correctly, we need to 
+track them. You can run the following command inside your model repo to do so.
+
+```
+cd norwegian-roberta-base
```

Comment:
This is a bit hacky, but it's there is no `git_lfs_track` abstraction in `transformers` yet. Think for the sprint, it's ok to leave the somewhat hacky solution for now.

Reply:
Alternatively you could do it like that with huggingface-hub v0.0.13:

```py
from huggingface_hub.repository import Repository
from transformers import BertModel, BertConfig

model = BertModel(BertConfig(num_hidden_layers=1, num_attention_heads=2, hidden_size=2, intermediate_size=2, vocab_size=10))

repo = Repository("repo-id", clone_from="<user>/repo-id", use_auth_token=True)

with repo.commit("Track tfevents"):
    repo.lfs_track("*tfevents*")

with repo.commit("New model version"):
    model.save_pretrained(".")
```

Reply:
going for the simple `git lfs track "*tfevents*"` solution for new. 

The user needs to pass the `"user-name"` which would require some changes to the scripts. Think we can do this in a follow-up PR.